socorro/processor/processor_pipeline.py

```diff
@@ -81,7 +81,7 @@ class ProcessorPipeline(RequiredConfig):
         "command_line",
         doc="template for the command to invoke the external program; uses Python format syntax",
         default=(
-            "timeout -s KILL {kill_timeout} {command_pathname} "
+            "timeout --signal KILL {kill_timeout} {command_pathname} "
```

Comment:
While I was fixing this bug, I figured I'd expand this argument to the long name so it's clearer.